testing/node_fetcher.ts

```diff
@@ -23,6 +21,14 @@ export interface BaseFetcherOptions {
   qs?: {[key: string]: any};
 
   followRedirect?: boolean;
+  /**
```

Comment:
This file is just copying over the latest from `coda` for the simulator, with a few minor changes to remove unneeded functionality. More comments on the `coda` PR about why these changes were necessary.